lab-carolina/lib/basic-auth.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+const createError = require('http-errors');
+
+module.exports = function(req, res, next){
+
+  var authHeader = req.headers.authorization;
+  if (!authHeader)
```

Comment:
Try not to get too fancy with omitting the curly brackets. Leave your return on one line if you'd like to omit them.